lib/main.dart

```diff
@@ -3,9 +3,15 @@ import 'package:firebase_core/firebase_core.dart';
 import 'package:tg_proj/misc/color_to_material_color.dart';
 import 'package:flutter/services.dart';
 import 'package:tg_proj/misc/router.dart';
+import 'package:geolocator/geolocator.dart';
+
+LocationPermission? usrLocPerm;
+
+get locPerm => usrLocPerm;
```

Comment:
Fajn, že se ti to podařilo vyřešit 😉. Ale tu proměnnou kde se to drží by to chtělo dát někam jinam než do mainu (třeba do nějakého singletonu). Pojďme ten main držet čistý 😁